test/system/700-play.bats

```diff
@@ -531,3 +531,68 @@ spec:
     run_podman pod rm -a -f
     run_podman rm -a -f
 }
+
+@test "podman kube play - multi-pod YAML" {
```

Comment:
Looks like you need `skip_if_remote`.

And, this entire test file has become unmanageable. I've got a refactor in progress, will submit PR after yours merges.

Reply:
Thank you.  I agree, there is a lot of redundant boilerplate in this file.  Some tests could also be merged but I never found the courage to tackle it.

Thanks for doing it!